packages/backend-core/src/constants/db.ts

```diff
@@ -21,7 +21,7 @@ export enum ViewName {
   USER_BY_EMAIL = "by_email2",
   BY_API_KEY = "by_api_key",
   LINK = "by_link",
-  ROUTING = "screen_routes",
+  ROUTING = "screen_routes_2",
```

Comment:
This will force all the screen routes views to be "recreated", as they will be considered as a new one